pkg/graph/resolvers/cedar_set_roles_for_user.go

```diff
@@ -51,8 +51,6 @@ func SetRolesForUserOnCEDARSystem(
 				appcontext.ZLogger(ctx).Error("failed to send email to user who was added to CEDAR system", zap.Error(err))
 				return nil, err
 			}
-		}
-		if rs.DidAdd {
```

Comment:
just curious why this is removed?

Reply:
I spoke with Jeff and both the leads email and the user email are only supposed to be sent when a user is newly added, not when roles are added.